src/toil/provisioners/aws/awsProvisioner.py

```diff
@@ -589,15 +617,17 @@ def _deleteIAMProfiles(cls, instances, ctx):
                 else:
                     raise
 
+    @awsRetry
+    def _propagateToNode(self, node):
+        # since we're going to be rsyncing into the appliance we need the appliance to be running first
+        ipAddress = self._waitForNode(node, 'toil_worker')
+        self._coreRsync(ipAddress, [self.config.sseKey, ':' + self.config.sseKey], applianceName='toil_worker')
+
     def _propagateKey(self, instances):
```

Comment:
Would a better name be _propagateKeyToNode and _propagateKey for the above function?